core/src/main/kotlin/transformers/documentation/DefaultDocumentationNodeMerger.kt

```diff
@@ -64,16 +64,34 @@ fun Property.mergeWith(other: Property) = Property(
     (actual + other.actual).merge()
 )
 
+fun Classlike.mergeWith(other: Classlike): Classlike = when {
+    this is Class && other is Class -> mergeWith(other)
+    this is Enum && other is Enum -> mergeWith(other)
+    else -> TODO("${this::class.qualifiedName} ${this.name} cannot be merged with ${other::class.qualifiedName} ${other.name}")
```

Comment:
Nice, only lacks a proper exception instead of this `TODO`

Reply:
Changed, it's an exception now.